app/controllers/rentals_controller.rb

```diff
@@ -19,6 +22,7 @@ def show
   # GET /rentals/new
   def new
     @rental = Rental.new
+    @start_date = params['start_date'] ? params['start_date'].to_date : Time.zone.today
```

Comment:
For a shorter line, you might be able to do `params['start_date'].try(:to_date) || Time.zone.today`